koku/providers/gcp/provider.py

```diff
@@ -40,6 +40,13 @@ def cost_usage_source_is_reachable(self, credential_name, data_source):
             key = 'billing_source.bucket'
             raise serializers.ValidationError(error_obj(key, e.message))
 
+        report_prefix = data_source.get('report_prefix', '')
+        if report_prefix:
+            if len(report_prefix) > 64:
+                key = 'billing_source.report_prefix'
+                message = f'report_prefix length is {len(report_prefix)}, while the maximum allowed is 64.'
```

Comment:
We could make this _look_ more like a standard DRF `CharField` validation error message (even though it's all custom validation) if we use the string:

```
'Ensure this field has no more than {max_length} characters.'
```